src/clr-angular/utils/drag-and-drop/providers/drag-and-drop-event-bus.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Injectable} from "@angular/core";
+import {Observable} from "rxjs/Observable";
+import {Subject} from "rxjs/Subject";
+import {ClrDraggableEvent} from "../interfaces/draggable-event";
+
+@Injectable()
+export class ClrDragAndDropEventBus {
```

Comment:
The term `eventbus` seems out of place for me. Its really just a service. Shouldn't we name it `ClrDragEventService`?

Reply:
@adityarb88 and @youdz bullied me and made me change this service's name. It was DragAndDropDispatcher initially. 😂 :trollface: 

Actually, this is the service class that coordinates events between draggables and droppables. So it carries events from draggables and transfer them to droppables. I guess that's why we've named it EventBus. 🚌 🤷‍♂️

Reply:
🎉 🚌